templates/controllers/modals/submission/viewSubmissionMetadata.tpl

```diff
@@ -0,0 +1,28 @@
+{**
+ * lib/pkp/templates/controllers/modals/submission/viewSubmissionMetadata.tpl
+ *
+ * Copyright (c) 2014-2019 Simon Fraser University
+ * Copyright (c) 2003-2019 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Display submission metadata.
+ *}
+
+<div id="viewSubmissionMetadata" class="">
+	<h3>{$title|escape}</h3>
+	{if $authors}<h4>{$authors|escape}</h4>{/if}
+	<div class="abstract">
+		{$abstract}
+	</div>
+	{if $additionalMetadata}
+		<table>
```

Comment:
You can add `class="pkpTable"` to align the styles with the new tables component from vue.

Reply:
check

Reply:
It looks like the class is still missing in this PR?

Reply:
checkcheck